### ISSUE TITLE
[FEATURE REQUEST] - duplicate the Macros

### ISSUE THREAD

**Issue by Zizipack** [chhoumann/quickadd#571]
Add the ability to duplicate the Macro with all fields, delays, and internal fields

Right now, you can create a duplicate of the shortcut from which the macro starts, which is useless
![image](https://github.com/chhoumann/quickadd/assets/83618118/1149ad4e-163e-4682-b6ac-27f080218e43)

Actually, it's necessary to copy the macro itself with all templates and their settings inside and everything else
![image](https://github.com/chhoumann/quickadd/assets/83618118/ddb044d3-d9b5-4d25-86c5-159a73562387)
![image](https://github.com/chhoumann/quickadd/assets/83618118/ffaa6913-a953-47bd-a216-1c3825efbd37)

**Comment by girishkdesh** [comment]
Yes this is an important feature if a second macro needs to be just like the first one and only has some changes in the template or capture format and so.

**Comment by gayoungmoney** [comment]
that's true, it just copies the name of macro, not all command inside

**Comment by Zizipack** [comment]
Why does this automation program for creating notes still require manually inputting macros again, even if only a small part of it has changed, and why is there still no convenient menu with all the macros?

**Comment by chhoumann** [comment]
Hi everyone! Good news - this feature was added in the recent 2.0.0 update. 

To duplicate a macro with all its commands, delays, and settings:

1. Open QuickAdd settings (Settings → QuickAdd)
2. Find your macro in the choice list
3. Click the **copy icon** (📋) next to your macro choice
4. A complete duplicate will be created with "(copy)" appended to the name

The duplicate includes everything:
- All commands (Template, Capture, Wait, AI commands, etc.)
- All delays and settings
- All internal configurations

Let me know if you need help finding the duplicate button or if you're experiencing any issues with the functionality!
